x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -54,6 +54,15 @@ export function InfraHostsViewProvider({ getService }: FtrProviderContext) {
       return testSubjects.find('hostsView-metricChart');
     },
 
+    getMetricsTab() {
+      return testSubjects.find('hostsView-tabs-metrics');
+    },
+
+    async visitMetricsTab() {
+      const alertsTab = await this.getMetricsTab();
```

Comment:
This naming looks a bit confusing - should it be `metricsTab`

Reply:
Whoops good catch, left over from copy paste a line 👍